pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "pyninjotiff"
+version = "0.3.0"
```

Comment:
Could this instead be automatically derived from git tags, similar to setuptools-scm?  I'm not at all familiar with poetry so I don't know its capabilities or limitations.  It does say at line 35 that dynamic versioning is enabled?